### PR TITLE
Improve segments admin visibility

### PR DESCRIPTION
**Pull request by bencrouse**
Show segments throughout the admin

### CODE REVIEW

File: admin/app/views/workarea/admin/orders/fraud.html.haml [hunk 1]
```diff
@@ -14,29 +14,30 @@
...
-              %tr
-                %th
-                  %span.data-pairs__name= t('workarea.admin.orders.fraud.response')
-                %td
-                  = @order.fraud_decision.response || '-'
-                  = link_to '#response-help', data: { tooltip: '' } do
-                    = inline_svg('workarea/admin/icons/help.svg', class: 'svg-icon svg-icon--small svg-icon--blue', title: t('workarea.admin.orders.fraud.more_link'))
-                  #response-help.tooltip-content
-                    %p= t('workarea.admin.orders.fraud.response_tooltip')
+    .section
```
Comment (tubbo) [inline admin/app/views/workarea/admin/orders/fraud.html.haml]: why this additional markup?

Comment (tubbo) [inline admin/app/views/workarea/admin/orders/fraud.html.haml]: why this additional markup?

Comment (tubbo) [inline admin/app/views/workarea/admin/orders/fraud.html.haml]: why this additional markup?

Comment (bencrouse) [reply admin/app/views/workarea/admin/orders/fraud.html.haml]: It fixes vertical spacing between the cards and content on this page.

**Pull request merged**